===================================================================
RCS file: /home2/webcvs/mirror/qt-copy/qmake/generators/unix/unixmake2.cpp,v
retrieving revision 1.29
retrieving revision 1.30
diff -u -p -r1.29 -r1.30
--- qt-copy/qmake/generators/unix/unixmake2.cpp	2003/08/27 14:31:43	1.29
+++ qt-copy/qmake/generators/unix/unixmake2.cpp	2003/08/27 16:40:35	1.30
@@ -1337,7 +1337,7 @@ UnixMakefileGenerator::libtoolFileName()
 void
 UnixMakefileGenerator::writeLibtoolFile()
 {
-    QString fname = pkgConfigFileName(), lname = fname;
+    QString fname = libtoolFileName(), lname = fname;
     int slsh = lname.findRev(Option::dir_sep);
     if(slsh != -1)
 	lname = lname.right(lname.length() - slsh);
